src/core/lib/event_engine/posix_engine/ev_epoll1_linux.cc

```diff
@@ -388,8 +390,11 @@ Epoll1Poller::~Epoll1Poller() {
       delete handle;
     }
   }
+  closed_ = true;
```

Comment:
In the rare occurrence that a Kick and a fork event happen at roughly the same time, there may be concurrent read/write on `closed_`. I think it'll work out if we guard the variable by `mu_` as well.

Reply:
Oh, duh. Fixed.